pages/release.rst

```diff
@@ -37,3 +36,15 @@ RELEASE	CAMERA FILTER     PRIMARY
 5000	wise   :math:`W3` 0
 5000	wise   :math:`W4` 0
 ======= ====== ========== =======
+
+======= ====== ========== =======
+RELEASE	CAMERA FILTER     PRIMARY
```

Comment:
The spacing in this table looks a bit weird.  Double check that that is not affecting the display of the table.